cmd/skaffold/app/cmd/dev.go

```diff
@@ -36,9 +36,16 @@ func NewCmdDev(out io.Writer) *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return dev(out)
 		},
+		Annotations: map[string]string{
+			BuildAnnotation:  "ture",
```

Comment:
nit: true

Reply:
maybe a 

```golang
map[string]struct{}
```
would make sense here if the string value isn't being used?

Reply:
Annotations is Builtin Field for cobra.Command Struct.